test/cmdlineTests/ir_compiler_subobjects/output

```diff
@@ -0,0 +1,498 @@
+IR:
+/*******************************************************
+ *                       WARNING                       *
+ *  Solidity to Yul compilation is still EXPERIMENTAL  *
+ *       It can result in LOSS OF FUNDS or worse       *
+ *                !USE AT YOUR OWN RISK!               *
+ *******************************************************/
+
+
+object "C_12" {
+    code {
+        mstore(64, 128)
+        if callvalue() { revert(0, 0) }
+
+        constructor_C_12()
+
+        codecopy(0, dataoffset("C_12_deployed"), datasize("C_12_deployed"))
+        return(0, datasize("C_12_deployed"))
+
```

Comment:
Is this rebased on top of latest develop?

Reply:
I will rebase at the end once more to make sure